src/duckdb_node.cpp

```diff
@@ -55,6 +55,29 @@ static Napi::Value ConvertStringVector(const Napi::CallbackInfo &info) {
 	return array;
 }
 
+static Napi::Value ConvertBooleanVector(const Napi::CallbackInfo &info) {
+	Napi::Env env = info.Env();
+	auto vector = duckdb_node::ValueConversion::FromJS<duckdb_vector>(info, 0);
+	auto n = duckdb_node::ValueConversion::FromJS<idx_t>(info, 1);
+
+	auto type_id = duckdb_get_type_id(duckdb_vector_get_column_type(vector));
```

Comment:
Hm, does this leak the duckdb_logical_type returned by duckdb_vector_get_column_type? (This is copied from ConvertStringVector above, so the question applies to that code as well.)

Reply:
This and ConvertStringVector are now removed in favor of doing the conversion in JS, which is measurably faster, so this question is moot.